### ISSUE TITLE
Earlier versions of camel-cmis only allow creation of cmis:document objectTypeIds 

### ISSUE THREAD

**Issue by vinny2020** [bibryam/camel-cmis#1]
In version 2.12.4 any attempt to create an object that extends cmis:document fails because the code in that version only allows cmis:documents to be created not their extentions. 

The code that does this resides in CMISSessionFacade: 
  `public boolean isObjectTypeVersionable(String objectType) {
        if (CamelCMISConstants.CMIS_DOCUMENT.equals(objectType))`

I believe that the more recent version of the camel-cmis component fixes this but I'm not using the latest and greatest camel and I'm wondering if the fix could be back ported. If you need help, I can fork and do a pull request .

**Comment by bibryam** [comment]
The policy for release branches is tha they only get bug fixes. This one is more of an imporvement.
May be temporarily, you can clone the component and use an modified version of it in your project. And later remove it once Camel is upgraded.

**Comment by vinny2020** [comment]
Good point. I tried using the latest version and confirmed that it did indeed work. Surprisingly maven did not pull in the latest camel-\* jars into my project , but I was grateful. This led to another issue which I'll post separately.
